### PR TITLE
{chem}[foss/2022a] CP2K v9.1, v2022.1

### PR DESCRIPTION
**Pull request by alinelena**
some tests are failing, see the comments in #15146

### CODE REVIEW

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ generoso

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/c/CP2K/CP2K-9.1-foss-2022a.eb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+toolchainopts = {'pic': True, 'openmp': True}
+
+source_urls = ['https://github.com/cp2k/cp2k/releases/download/v%(version)s.0/']
+sources = [SOURCELOWER_TAR_BZ2]
+checksums = ['fedb4c684a98ad857cd49b69a3ae51a73f85a9c36e9cb63e3b02320c74454ce6']
+
+dependencies = [
+    ('Libint', '2.6.0', '-lmax-6-cp2k'),
```
Comment (boegel) [inline easybuild/easyconfigs/c/CP2K/CP2K-9.1-foss-2022a.eb:21]: @alinelena Is there a strict need for Libint 2.6.0 here?
See also #16073 ...

Comment (alinelena) [reply easybuild/easyconfigs/c/CP2K/CP2K-9.1-foss-2022a.eb:21]: nope if 2.7.1 or whatever is available shall be used instead.

File: easybuild/easyconfigs/c/CP2K/CP2K-2022.1-foss-2022a.eb [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+toolchain = {'name': 'foss', 'version': '2022a'}
+toolchainopts = {'pic': True, 'openmp': True}
+
+source_urls = ['https://github.com/cp2k/cp2k/releases/download/v%(version)s/']
+sources = [SOURCELOWER_TAR_BZ2]
+checksums = ['2c34f1a7972973c62d471cd35856f444f11ab22f2ff930f6ead20f3454fd228b']
+
+#                https://github.com/cp2k/cp2k/releases/download/v2022.1/cp2k-2022.1.tar.bz2
+dependencies = [
+    ('Libint', '2.6.0', '-lmax-6-cp2k'),
```
Comment (boegel) [inline easybuild/easyconfigs/c/CP2K/CP2K-2022.1-foss-2022a.eb:22]: @alinelena Please update to 2.7.2 (see #16073), and remove the Libint easyconfig file from this PR

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by maxim-masterov** [comment]
Can this PR be updated?

File: easybuild/easyconfigs/c/CP2K/CP2K-9.1-foss-2022a.eb (hunk 1 above)
Comment (lexming) [reply easybuild/easyconfigs/c/CP2K/CP2K-9.1-foss-2022a.eb]: fixed in https://github.com/easybuilders/easybuild-easyconfigs/pull/16007/commits/8d95a7d96fd1ed37c1752c5830ee327c1a9ed95d

File: easybuild/easyconfigs/c/CP2K/CP2K-2022.1-foss-2022a.eb (hunk 2 above)
Comment (lexming) [reply easybuild/easyconfigs/c/CP2K/CP2K-2022.1-foss-2022a.eb]: fixed in https://github.com/easybuilders/easybuild-easyconfigs/pull/16007/commits/8d95a7d96fd1ed37c1752c5830ee327c1a9ed95d

**Comment by lexming** [comment]
@boegelbot please test @ generoso
CORE_CNT=16

**Comment by lexming** [comment]
@boegelbot please test @ jsc-zen2
CORE_CNT=16

**Review by lexming — APPROVED** [review]
LGTM

**Comment by lexming** [comment]
Thanks @alinelena , going in!

**Pull request merged**